### PR TITLE
Release preparations

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: doc/Changelog.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
 # Changelog
 
 
-# git master {#master}
+## git master {#master}
```
Comment (tribal-tec) [inline doc/Changelog.md]: either in doxygen or on github the double hashes are not rendered right. Why needed? Remove the first line?

Comment (eile) [inline doc/Changelog.md]: Gah! github looks ok, checking doxygen.

Comment (eile) [inline doc/Changelog.md]: Needed since it's a subheading to the ChangeLog heading.

**Comment by eile** [comment]
Updated

**Pull request merged**